cogs/ReactForRole.py

```diff
@@ -1273,6 +1315,41 @@ def get_guild_rfr_required_roles(self, guild_id) -> List[int]:
             return []
         return role_ids
 
+    def add_guild_inline_status(self, guild_id):
+        """
+        Add an inline status for all rfr messages in a guild
+        :param guild_id: guild ID
+        """
+        self.database_manager.db_execute_commit(
+            "INSERT INTO InlineALlStatus (guild_id, inline_status) VALUES (?, ?);",
+            args=[guild_id, "N"])
+
+    def update_guild_inline_status(self, guild_id, status):
+        """
+        Updates the inline status for all rfr messages in a guild
+        :param guild_id: guild ID
+        :param status: inline all status "Y" or "N"
+        """
+        self.database_manager.db_execute_commit(
+            "UPDATE InlineAllStatus SET inline_status = ? WHERE guild_id = ?;", args=[status, guild_id])
+
+    def get_guild_inline_status(self, guild_id):
+        """
+        Gets the inline status for the specific guild
+        :param guild_id: guild ID
+        :return: Inline All Status for that guild
+        """
+        return self.database_manager.db_execute_select("SELECT inline_status FROM InlineALlStatus WHERE guild_id = ?",
```

Comment:
Noticed you have "InlineALlStatus", I think you accidentally capitalised the first L in All

Reply:
Fixed this typo